Packs/Troubleshoot/IncidentFields/incidentfield-Instance_Name.json

```diff
@@ -0,0 +1,34 @@
+{
+ "associatedToAll": true,
+ "caseInsensitive": true,
+ "cliName": "instancename",
+ "closeForm": false,
+ "content": true,
+ "description": "Name of the instance to troubleshoot",
+ "editForm": true,
+ "fieldCalcScript": "096f78c3-e8a5-42ee-82e5-1540fc285d3f",
```

Comment:
this script id should be removed i believe

Reply:
you believe wrong.